<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>RGB</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="RS">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="GS">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="BS">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comL"/>
      </item>
      <item>
       <widget class="QPushButton" name="openB">
        <property name="text">
         <string>OPEN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeB">
        <property name="text">
         <string>CLOSE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDial" name="servoK">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>SERVO</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="tempB">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>31</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimum">
     <number>200</number>
    </property>
    <property name="maximum">
     <number>400</number>
    </property>
    <property name="value">
     <number>199</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tempL">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdN">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="circle">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="ledC">
       <property name="text">
        <string>LED</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fanC">
       <property name="text">
        <string>FAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bulbC">
       <property name="text">
        <string>BULB</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="textF"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendB">
       <property name="text">
        <string>SEND</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
